components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/event/OAuthEventInterceptor.java

```diff
@@ -229,6 +229,30 @@ default void onPostTokenRevocationBySystem(AccessTokenDO accessTokenDO, Map<Stri
 
     }
 
+    /**
+     *
+     * This will be called before when Tokens Revoked through Listeners directly.
```

Comment:
Also, the purpose of this method is not clear from its name. Can you explain when this method should be called?